aws-logs-resourcepolicy/src/test/java/software/amazon/logs/resourcepolicy/AbstractTestBase.java

```diff
@@ -0,0 +1,55 @@
+package software.amazon.logs.resourcepolicy;
+
+import software.amazon.awssdk.awscore.AwsRequest;
+import software.amazon.awssdk.awscore.AwsResponse;
+import software.amazon.awssdk.core.pagination.sync.SdkIterable;
+import software.amazon.awssdk.services.cloudwatchlogs.CloudWatchLogsClient;
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
+import software.amazon.cloudformation.proxy.Credentials;
+import software.amazon.cloudformation.proxy.LoggerProxy;
+import software.amazon.cloudformation.proxy.ProxyClient;
+
+import java.util.Arrays;
+import java.util.concurrent.CompletableFuture;
+import java.util.function.Function;
+
+public class AbstractTestBase {
+  protected static final Credentials MOCK_CREDENTIALS;
+  protected static final LoggerProxy logger;
+
+  static {
+    MOCK_CREDENTIALS = new Credentials("accessKey", "secretKey", "token");
+    logger = new LoggerProxy();
+  }
+  static ProxyClient<CloudWatchLogsClient> MOCK_PROXY(
```

Comment:
see comment earlier about using injectCredentialsAndInvokeV2 instead of the describe call. I think you'd be able to remove this whole class because you wouldn't have to mock this proxy client